src/client/components/filter-tile/filter-tile.tsx

```diff
@@ -27,21 +28,36 @@ export interface ItemBlank {
   clause?: FilterClause;
 }
 
-function formatLabel(dimension: Dimension, clause: FilterClause, essence: Essence): string {
+export interface LabelFormatOptions {
+  dimension: Dimension;
+  clause: FilterClause;
+  essence: Essence;
+  verbose?: boolean;
+}
+
+export function formatLabel(options: LabelFormatOptions): string {
```

Comment:
rather than exporting a function from a component it should be moved into a shared util